config/vaccines.yml

```diff
@@ -34,6 +34,50 @@ vaxigrip:
   snomed_product_term: Vaxigrip Trivalent influenza vaccine (split virion,
     inactivated) suspension for injection 0.5ml pre-filled syringes (Sanofi)
 
+viatris:
+  brand: Viatris
+  type: flu
+  method: injection
+  manufacturer: Viatris
+  nivs_name: Viatris
```

Comment:
Not sure about the `nivs_name` for all products.

Reply:
The `nivs_name` isn't too important for flu as we don't allow importing historical records.